standards/rmrk2.0.0/interactions/mint.md

```diff
@@ -0,0 +1,33 @@
+# MINT
+
+The MINT interaction creates an [NFT](../entities/nft.md) in/from a [class](../entities/nftclass.md).
+
+## Standard
+
+The format of a MINT interaction is the same as for [CREATE](create.md):
+`0x{bytes(rmrk::MINT::{version}::{html_encoded_json})}`.

```

Comment:
Please add example of interaction remark when NFT is Minted and another NFT is it's owner